### DISCUSSION TITLE
Is it possible to edit the magic number?

### DISCUSSION THREAD

**Discussion by Blank-c** [pyinstaller/pyinstaller#6959] [Help] [answered]
It is to prevent the code from being unpacked with `pyintxtractor`? I know it can be unpacked with some changes in the code but still is there a way?

**Comment by rokm** [comment, marked as answer]
The magic number is hardcoded in both pre-compiled bootloader ([here](https://github.com/pyinstaller/pyinstaller/blob/436129e41f988ced238f55aecb3caf7bf0f1a74a/bootloader/src/pyi_utils.c#L1351) and [here](https://github.com/pyinstaller/pyinstaller/blob/436129e41f988ced238f55aecb3caf7bf0f1a74a/bootloader/src/pyi_main.c#L66-L67)) and on the python side of the PyInstaller code (the archive writer - [here](https://github.com/pyinstaller/pyinstaller/blob/436129e41f988ced238f55aecb3caf7bf0f1a74a/PyInstaller/archive/writers.py#L282)), so there is no way of changing it dynamically.

That said, the code is out there, so you're free to modify it however you want. Just don't forget to compile the bootloader, and don't expect any support.
